test/liquidation/makeLiquidatableProtocol.ts

```diff
@@ -54,75 +54,75 @@ export default async function makeLiquidatableProtocol() {
     extensionDelegate: extensionDelegate.address,
     baseToken: USDC,
     baseTokenPriceFeed: USDC_USD_PRICE_FEED,
-    supplyKink: 800000000000000000n, // .8e18
+    supplyKink: exp(0.8, 18),
     supplyPerYearInterestRateBase: 0n,
-    supplyPerYearInterestRateSlopeLow: 50000000000000000n, // .05e18
-    supplyPerYearInterestRateSlopeHigh: 2000000000000000000n, // 2e18
-    borrowKink: 800000000000000000n, // .8e18
-    borrowPerYearInterestRateBase: 5000000000000000n, // .005e18
-    borrowPerYearInterestRateSlopeLow: 100000000000000000n, // .1e18
-    borrowPerYearInterestRateSlopeHigh: 3000000000000000000n, // 3e18
-    storeFrontPriceFactor: 1000000000000000000n,
-    trackingIndexScale: 1000000000000000n,
-    baseTrackingSupplySpeed: 1000000000000000n,
-    baseTrackingBorrowSpeed: 1000000000000000n,
-    baseMinForRewards: 1000000n,
-    baseBorrowMin: 1000000n,
-    targetReserves: 1000000000000000000n,
+    supplyPerYearInterestRateSlopeLow: exp(0.5, 18),
+    supplyPerYearInterestRateSlopeHigh: exp(2, 18),
+    borrowKink: exp(0.8, 18),
+    borrowPerYearInterestRateBase: exp(5, 15),
+    borrowPerYearInterestRateSlopeLow: exp(0.1, 18),
+    borrowPerYearInterestRateSlopeHigh: exp(3, 18),
+    storeFrontPriceFactor: exp(1, 18),
+    trackingIndexScale: exp(1, 15),
+    baseTrackingSupplySpeed: exp(1, 15),
+    baseTrackingBorrowSpeed: exp(1, 15),
+    baseMinForRewards: exp(1, 6),
+    baseBorrowMin: exp(1, 6),
+    targetReserves: exp(1, 18),
     assetConfigs: [
       {
         asset: DAI,
         priceFeed: DAI_USDC_PRICE_FEED,
         decimals: 18,
         borrowCollateralFactor: 999999999999999999n,
-        liquidateCollateralFactor: 1000000000000000000n,
-        liquidationFactor: 900000000000000000n,
-        supplyCap: 1000000000000000000000000n
+        liquidateCollateralFactor: exp(1, 18),
+        liquidationFactor: exp(0.9, 18),
+        supplyCap: exp(1000000, 18)
       },
       {
         asset: COMP,
         priceFeed: COMP_USDC_PRICE_FEED,
         decimals: 18,
         borrowCollateralFactor: 999999999999999999n,
-        liquidateCollateralFactor: 1000000000000000000n,
-        liquidationFactor: 900000000000000000n,
-        supplyCap: 100000000000000000000n
+        liquidateCollateralFactor: exp(1, 18),
+        liquidationFactor: exp(0.9, 18),
+        supplyCap: exp(100, 18)
       },
       {
         asset: WBTC,
         priceFeed: WBTC_USDC_PRICE_FEED,
         decimals: 8,
         borrowCollateralFactor: 999999999999999999n,
-        liquidateCollateralFactor: 1000000000000000000n,
-        liquidationFactor: 900000000000000000n,
-        supplyCap: 1000000000000000000000000n
+        liquidateCollateralFactor: exp(1, 18),
+        liquidationFactor: exp(0.9, 18),
+        supplyCap: exp(1000, 8)
```

Comment:
wouldn't the equivalent be `exp(10000000000000000, 8)` (or `exp(1000000, 18)`)?

Reply:
I changed the supply cap to be more realistic since there are only 10 millions BTC in existence and it has 8 decimals scale, so 1000000000000000000000000 looks extreme